server/sonar-db-dao/src/main/resources/org/sonar/db/qualityprofile/QProfileEditUsersMapper.xml

```diff
@@ -18,12 +18,70 @@
     and qeu.qprofile_uuid = #{qProfileUuid, jdbcType=VARCHAR}
   </select>
 
+  <select id="countByQuery" resultType="int">
+    select count(u.login)
+    <include refid="sqlSelectByQuery" />
+  </select>
+
+  <select id="selectByQuery" parameterType="map" resultType="org.sonar.db.qualityprofile.QProfileEditUserMembershipDto">
```

Comment:
should we implement db specific pagination systematically? Because I don't expect this table to have high volumes

Reply:
Yes, it's the only good way to do pagination for the moment. It has been validated by Simon